client/src/components/PageComponents/Header/Header.tsx

```diff
@@ -8,21 +9,67 @@ import {
 } from "./Header.css";
 
 const Header = () => {
+  const navigate = useNavigate();
+
+  const gotoMainPage = () => {
+    navigate("/");
+  };
+
+  const gotoSchedulePage = () => {
+    navigate("/schedule");
+  };
+
+  const gotoCalendarPage = () => {
+    navigate("/calendar");
+  };
+
+  const gotoTodolistPage = () => {
+    navigate("/todolist");
+  };
+
+  /* 모달들은 스타일을 확인할 수 있게 새 페이지로 연결만 시켜놓았습니다. */
+  const openLoginModal = () => {
+    navigate("/loginmodal");
```

Comment:
임시 경로인건 알고 있지만 이런 경우에도 컨벤션을 지켜서 작성해주는 습관을 들이시면 좋습니다~
그리고 URL 경로에서는 카멜케이스가아닌 케밥케이스가 더 좋습니다!
```suggestion
    navigate("/login-modal");
```